src/styles/main.scss

```diff
@@ -1,3 +1,203 @@
+* {
+  box-sizing: border-box;
+}
+
+p {
+  margin: 0;
+}
+
+$button-bg-color: #00acdc;
+$main-color-text: #060b35;
+$main-light-text: #616070;
+
 body {
+  font-family: "Roboto", sans-serif;
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px ;
+  &__logo {
+    margin-top: 10px;
+    width: 40px;
+    height: 40px;
+    &-img {
+      width: 100%;
+      height: 100%;
+    }
+  }
+}
+
+.nav {
+  padding: 10px 0 20px;
+  &__list {
+    margin-top: 0;
+    margin-bottom: 4px;
+    padding-top: 9px;
+    padding-left: 0;
+    list-style: none;
+    display: flex;
+  }
+  &__item {
+    position: relative;
+    margin-left: 20px;
+    &:first-child {
+      margin-left: 0;
+    }
+  }
+  &__link {
+    color: $main-color-text;
+    font-size: 12px;
+    text-decoration: none;
+    font-weight: 500;
+    text-transform: uppercase;
+    padding: 23px 0;
+    &:hover {
+      color: #00acdc;
+    }
+  }
+  
+  .is-active {
+    color: #00acdc;
+    &::after {
+      content: "";
+      position: absolute;
+      bottom: -22px;
+      left: 0;
+      height: 4px;
+      width: 100%;
+      background: #00acdc;
+      border-radius: 8px;
+    }
+  }
+}
+
+.main {
+  margin-top: -2px;
+}
+
+.cards {
+  display: flex;
+  max-width: 944px;
```

Comment:
<img width="989" alt="image" src="https://user-images.githubusercontent.com/102411130/220049076-9aa3ee42-c7ad-46a1-9a92-8479e1e7ea10.png">
width, position, margins are external styles

Reply:
check the previous comment
don't set external styles to bem block, you can use mix

it can be look like that
```
<main class="main">
      <section class="main__cards cards">
      
.main {
  &__cards {
    move here your external styles(width, margins and so on)
    }
}
```